web/client/libs/ajax.js

```diff
@@ -10,6 +10,52 @@ var axios = require('axios');
 var ConfigUtils = require('../utils/ConfigUtils');
 var toString = Object.prototype.toString;
 
+const SecurityUtils = require('../utils/SecurityUtils');
+
+/**
+ * Internal helper that adds an extra paramater to an axios configuration.
+ */
+function addParameterToAxiosConfig(axiosConfig, parameterName, parameterValue) {
+    // FIXME: the parameters can also be a URLSearchParams
+    axiosConfig.params = Object.assign({}, axiosConfig.params, {[parameterName]: parameterValue});
```

Comment:
Object.assign is not compatible with all browsers, please use `require('object-assign)`